/**:
    ros__parameters:
      video_device: "/dev/video4"
      framerate: 30.0
      io_method: "mmap"
      frame_id: "camera"
      pixel_format: "yuyv"
      color_format: "yuv422p"
      image_width: 640
      image_height: 480
      camera_name: "test_camera"
      camera_info_url: "package://usb_cam/config/camera_info.yaml"
